src/Command/RecipesCommand.php

```diff
@@ -0,0 +1,253 @@
+
+                continue;
+            }
+
+            $operations[] = new InformationOperation($pkg);
+        }
+
+        $recipes = $this->flex->fetchRecipes($operations);
+        ksort($recipes);
+
+        $nbRecipe = \count($recipes);
+        if ($nbRecipe <= 0) {
+            $this->getIO()->writeError('<error>No recipe found</error>');
+
+            return 1;
+        }
+
+        // Display the information about a specific recipe
+        if (1 === $nbRecipe) {
+            $this->displayPackageInformation(current($recipes));
+
+            return 0;
+        }
+
+        // display a resume of all packages
+        $write = [
+            '',
+            '<bg=blue;fg=white>                      </>',
+            '<bg=blue;fg=white> Recipes available.   </>',
+            '<bg=blue;fg=white>                      </>',
+            '',
+        ];
+
+        /** @var Recipe $recipe */
+        foreach ($recipes as $name => $recipe) {
+            $lockRef = $this->symfonyLock->get($name)['recipe']['ref'] ?? null;
+
+            $additional = '';
+            if (null === $lockRef && null !== $recipe->getRef()) {
+                $additional = '<comment>(recipe not installed)</comment>';
+            } elseif ($recipe->getRef() !== $lockRef) {
+                $additional = '<comment>(update available)</comment>';
+            }
+            $write[] = sprintf(' * %s %s', $name, $additional);
+        }
+
+        $write[] = '';
+        $write[] = '<fg=blue>Run</>:';
+        $write[] = ' * composer symfony:recipes vendor/package to see details about a recipe.';
+        $write[] = ' * composer symfony:recipes:install vendor/package --force -v to update that recipe.';
+        //$write[] = ' * composer symfony:recipes:blame to see a tree of all of the installed/updated files.';
+        $write[] = '';
+
+        $this->getIO()->write($write);
+
+        return 0;
+    }
+
+    private function displayPackageInformation(Recipe $recipe)
+    {
+        $recipeLock = $this->symfonyLock->get($recipe->getName());
+
+        $lockRef = $recipeLock['recipe']['ref'] ?? null;
+        $lockFiles = $recipeLock['files'] ?? null;
+
+        $status = '<comment>up to date</comment>';
+        if ($recipe->isAuto()) {
+            $status = '<comment>auto-generated recipe</comment>';
+        } elseif (null === $lockRef && null !== $recipe->getRef()) {
+            $status = '<comment>recipe not installed</comment>';
+        } elseif ($recipe->getRef() !== $lockRef) {
+            $status = '<comment>update available</comment>';
+        }
+
+        $io = $this->getIO();
+        $io->write('<info>name</info>     : '.$recipe->getName());
+        $io->write('<info>version</info>  : '.$recipeLock['version']);
+        if (!$recipe->isAuto()) {
+            $io->write('<info>repo</info>     : '.sprintf('https://%s/tree/master/%s/%s', $recipeLock['recipe']['repo'], $recipe->getName(), $recipeLock['version']));
```

Comment:
It would be awesome to make this clickable: https://symfony.com/blog/new-in-symfony-4-3-console-hyperlinks

We might need to determine somehow if this feature is available / if `symfony/console` 4.3 is available.

Reply:
I agree but I feel like it doesn't look like it :/
And we are dependent on composer on this side

Reply:
indeed - even though composer allows newer versions, it looks like they still package an old version https://github.com/composer/composer/blob/1.9.1/composer.lock